charts/node-red/README.md

```diff
@@ -1,107 +1,77 @@
-# Node-RED
+# Node-Red
 
-Low-code programming for event-driven applications
-
-**This chart is not maintained by the Node-RED project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new)**
```

Comment:
This was actually requested by Node-RED here: https://discourse.nodered.org/t/node-red-helm-chart/3784/3 (even though it's been 2 years now...)